SRC/NBASIC/nbas111e.txt

```diff
@@ -4,63 +4,63 @@ By Nestor Soriano (Konami Man), December 2.004
 
 1. WHAT IS NESTORBASIC?
 
-NestorBASIC is a set of machine code routines, integrated in a single file. 
+NestorBASIC is a set of machine code routines, integrated in a single file.     
 It is intended for being used within MSX-BASIC programs. Without losing 
 Turbo-BASIC compatibility, NestorBASIC gives you the following capabilities:
 
-- Full access to all te available memory of the computer (all existing memory 
-  in the case of DOS 1, all free memory in the case of DOS 2), up to 4 Mb.
-- Access to VRAM, with data block exchange between VRAM and between RAM and 
+- Full access to all the available memory of the computer (all existing memory  
+in case of DOS 1, all free memory in the case of DOS 2), up to 4 Mb.

```

Comment:
A nitpick but could you please indent this line by two spaces as the other ones? 🙂

Reply:
Ok.